tests/runtime/engine/runner.py

```diff
@@ -519,13 +519,16 @@ def print_befores_and_after_output():
             print(warn("[   SKIP   ] ") + label)
             return Runner.SKIP_AOT_NOT_SUPPORTED
 
-        if p and p.returncode != test.return_code and not test.will_fail and not timeout:
-            print(fail("[  FAILED  ] ") + label)
-            print('\tCommand: ' + bpf_call)
-            print('\tUnclean exit code: ' + str(p.returncode))
-            print('\tOutput: ' + to_utf8(output))
-            print_befores_and_after_output()
-            return Runner.FAIL
+        if p:
```

Comment:
I feel like it's less confusing to just disallow the use of the `RETURN_CODE` directive and the `WILL_FAIL` directive e.g. here is our one test that uses `RETURN_CODE`

```
NAME exit code
PROG BEGIN { exit(69); }
RETURN_CODE 69
TIMEOUT 1
```
If we add `WILL_FAIL`, it fails the test and gets this error:
```
Unexpected exit code: 69 (expected != 69)
```

Reply:
Yeah makes sense. Let's keep it NAND